src/index.js

```diff
@@ -0,0 +1,58 @@
+import './style.css';
+
+const array = [
+  {
+    description: 'Shopping',
+    completed: false,
+    index: 0,
+  },
+  {
+    description: 'Going to school',
+    completed: false,
+    index: 0,
+  },
+  {
+    description: 'Washing the dishes',
+    completed: false,
+    index: 0,
+  },
+];
```

Comment:
- [ ] Kindly the `index` property should be different from one to another, so you can use it after that to delete or update a specific task with its unique index.

Reply:
Hi @keroloussamy. Thanks for reviewing my project. I did not get you totally. Can i send you direct message via slack to clarify your suggestion ?

Reply:
@fed1k Yes sure.